### PR TITLE
NRL: set input units of overall sensitivity to input units of first stage

### PR DESCRIPTION
**Pull request by trichter**
When using an accelerometer in NRL.get_response the input unit of the resulting response object is 'M/S'. I think it should be 'M/S**2'.

```py
from obspy.clients.nrl import NRL
nrl = NRL()
sensor = ['Guralp', 'CMG-5', '4G_10V', '100 Hz']
datalogger = ['REF TEK', 'RT 130 & 130-SMA', '1', '100']
response = nrl.get_response(datalogger, sensor)
print(response)

Channel Response
	From M/S (Velocity in Meters per Second) to COUNTS (Digital Counts)
	Overall Sensitivity: 1.00856e+06 defined at 1.000 Hz
	11 stages:
		Stage 1: PolesZerosResponseStage from M/S**2 to V, gain: 0.254929
		Stage 2: ResponseStage from V to V, gain: 1
		Stage 3: CoefficientsTypeResponseStage from V to COUNTS, gain: 629129
		Stage 4: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
		Stage 5: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
		Stage 6: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
		Stage 7: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
		Stage 8: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
		Stage 9: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
		Stage 10: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
		Stage 11: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
```

With this PR the output is

```
Channel Response
	From M/S**2 (Velocity in Meters per Second) to COUNTS (Digital Counts)
	Overall Sensitivity: 160518 defined at 1.000 Hz
	11 stages:
		Stage 1: PolesZerosResponseStage from M/S**2 to V, gain: 0.254929
		Stage 2: ResponseStage from V to V, gain: 1
		Stage 3: CoefficientsTypeResponseStage from V to COUNTS, gain: 629129
		Stage 4: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
		Stage 5: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
		Stage 6: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
		Stage 7: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
		Stage 8: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
		Stage 9: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
		Stage 10: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
		Stage 11: CoefficientsTypeResponseStage from COUNTS to COUNTS, gain: 1
```

### CODE REVIEW

**Comment by heavelock** [comment]
I think you forgot about setting the `input_units_description` along with `input_units`. In effect in results there is ``From M/S**2 (Velocity in Meters per Second)`` instead of ``M/S**2 (Acceleration in Meters Per Second Per Second)``.

File: obspy/clients/nrl/client.py [hunk 1]
```diff
@@ -216,7 +216,9 @@ def get_response(self, datalogger_keys, sensor_keys):
         # Combine both by replace stage one in the data logger with stage
         # one of the sensor.
         dl_resp.response_stages.pop(0)
-        dl_resp.response_stages.insert(0, sensor_resp.response_stages[0])
+        sensor_stage0 = sensor_resp.response_stages[0]
+        dl_resp.response_stages.insert(0, sensor_stage0)
+        dl_resp.instrument_sensitivity.input_units = sensor_stage0.input_units
```
Comment (heavelock) [inline obspy/clients/nrl/client.py]:
```suggestion
        dl_resp.instrument_sensitivity.input_units = sensor_stage0.input_units
        dl_resp.instrument_sensitivity.input_units_description = sensor_stage0.input_units_description
```

**Comment by trichter** [comment]
Thanks @heavelock for the suggested change and for introducing me to this fancy new github feature.

**Comment by megies** [comment]
Looks good to me..

**Comment by megies** [comment]
Merging this now, if there's any complaints please file another issue.

**Comment by krischer** [comment]
Sorry for being late to the party. Also looks good to me :+1: Adding an additional test might still be nice but the unit wrangling stuff is tested in other places so if does not break anything else I'm fine with it.

**Pull request merged**